packages/amplify-cli/src/extensions/amplify-helpers/resource-status.ts

```diff
@@ -321,7 +337,11 @@ export async function showResourceTable(category, resourceName, filteredResource
   const { table } = print;
 
   table(tableOptions, { format: 'markdown' });
+  if (tagsUpdated) {
+    print.info('Resource Tags Update Detected');
+  }
+  const changedResourceCount =
+    resourcesToBeCreated.length + resourcesToBeUpdated.length + resourcesToBeDeleted.length + tagsUpdated ? 1 : 0;
 
-  const changedResourceCount = resourcesToBeCreated.length + resourcesToBeUpdated.length + resourcesToBeDeleted.length;
   return changedResourceCount;
```

Comment:
Why are we only returning 1 or 0?

If we are only returning 1 or 0, should the variable name still be `changedResourceCount`?

Reply:
Tags is a single aspect either it's modified or not, Tags also doesn't fit in to the whole category thing

Reply:
I was concerned because the method is called elsewhere, but it doesn't look like anything uses the returned count besides evaluating it to be truthy or falsey.

I suggest returning a boolean in this case:

```ts
return resourcesToBeCreated.length + resourcesToBeUpdated.length + resourcesToBeDeleted.length + tagsUpdated > 0;
```

Reply:
https://repl.it/repls/LimitedWanRecords#index.js
I don't think we should mix types ((number + number + boolean))

Reply:
Oh sorry, I thought tagsUpdated was the number of tags updated.

In that case, maybe this would work:
```ts
return resourcesToBeCreated.length + resourcesToBeUpdated.length + resourcesToBeDeleted.length > 0 || tagsUpdated;
```